apps/nextjs-website/src/components/templates/CaseHistoryTemplate/CaseHistoryPageTemplate.tsx

```diff
@@ -0,0 +1,97 @@
+'use client';
+import React from 'react';
+import { Product } from '@/lib/types/product';
+import { Part } from '@/lib/types/part';
+import PartRenderer from '@/components/molecules/PartRenderer/PartRenderer';
+import { useTranslations } from 'next-intl';
+import ProductsShowcase from '@/components/organisms/ProductsShowcase/ProductsShowcase';
+import ProductBreadcrumbs from '@/components/atoms/ProductBreadcrumbs/ProductBreadcrumbs';
+import { pageToBreadcrumbs } from '@/helpers/breadcrumbs.helpers';
+import { Box, Typography, useTheme } from '@mui/material';
+
+export type CaseHistoryPageTemplateProps = {
+  slug: string;
+  title: string;
+  description?: string;
+  products: Product[];
+  parts: Part[];
+};
+
+const CaseHistoryPageTemplate = ({
+  slug,
+  title,
+  description,
+  products,
+  parts,
+}: CaseHistoryPageTemplateProps) => {
+  const { palette, spacing } = useTheme();
+  const t = useTranslations();
+
+  const ItemContainer = ({ children }: { children: React.ReactNode }) => (
+    <Box paddingBottom={spacing(10)} paddingX={{ xs: '2rem', md: '20rem' }}>
+      <Box maxWidth={1200}>{children}</Box>
+    </Box>
+  );
+
+  return (
+    <>
+      <ItemContainer>
+        <Box sx={{ marginTop: 10, paddingTop: 3, paddingBottom: spacing(10) }}>
+          <ProductBreadcrumbs
+            breadcrumbs={[
+              ...pageToBreadcrumbs('solutions', [
+                {
+                  name: title,
+                  path: slug,
+                },
+              ]),
+            ]}
+          />
+        </Box>
+        <Typography color={'text.primary'} variant='h4'>
+          {title}
+        </Typography>
+        {description && (
+          <Typography
+            component={typeof description === 'string' ? 'p' : 'div'}
+            color={'text.primary'}
+            variant='body1'
+          >
+            {description}
+          </Typography>
+        )}
+      </ItemContainer>
+      {parts.map((part: Part, index: number) =>
+        part.component !== 'quote' ? (
+          <ItemContainer key={index}>
+            <PartRenderer part={part} />
+          </ItemContainer>
+        ) : (
+          <Box
+            key={index}
+            sx={{
+              marginBottom: 6,
+            }}
+          >
+            <PartRenderer part={part} />
+          </Box>
+        )
+      )}
+      <ItemContainer>
+        <ProductsShowcase
```

Comment:
Pay attention to the distance between the title and the cards:
**Figma**
![image](https://github.com/pagopa/developer-portal/assets/39835990/372a8342-0f09-4e1e-940b-d4b99483814d)

**Storybook**
![image](https://github.com/pagopa/developer-portal/assets/39835990/2611f647-2fa2-4404-b349-3e4e9e5d87fe)